firebase/src/core/domain/use-cases/collections/sync-collections.ts

```diff
@@ -83,11 +83,19 @@ export class SyncCollectionsUseCase {
       // Adding/Updating collection
       //
       const memosOrder = localCollection.memos.map(({ id }) => id);
-      const storableCollection = {
-        ...localCollection,
+      const storableCollection: StoredPublicCollection = {
+        id: localCollection.id,
+        name: localCollection.name,
+        description: localCollection.description,
+        tags: localCollection.tags,
+        category: localCollection.category,
+        locale: localCollection.locale,
+        contributors: localCollection.contributors,
+        resources: localCollection.resources,
         memosAmount: memosOrder.length,
         memosOrder: memosOrder,
       };
+
```

Comment:
Had to update removing the spread operator because using like `...localCollection` adds `memos` property to the new object, which shouldn't have the memos.